packages/bitcore-lib-btg/lib/block/blockheader.js

```diff
@@ -9,7 +9,7 @@ var Hash = require('../crypto/hash');
 var JSUtil = require('../util/js');
 var $ = require('../util/preconditions');
 
-var GENESIS_BITS = 0x1f07ffff;
+var GENESIS_BITS = 0x1d00ffff;
```

Comment:
In BTG, the pre-fork bits is 0x1d00ffff (same as BTC), post-fork bits is 0x1f07ffff. Should check how large the change is.

Reply:
This is genesis_bits. For the blocks, it has bits param and difficulty is calculated with it so this variable doesn't affect to other blocks. @h4x3rotab

Reply:
Ah, you are correct!